File: employee-service/src/main/java/com/java/employee/service/EmployeeService.java
package com.java.employee.service;

import com.java.employee.dto.APIResponseDto;
import com.java.employee.dto.EmployeeDto;

public interface EmployeeService {

    EmployeeDto saveEmployee(EmployeeDto employeeDto);

    APIResponseDto getEmployeeById(Long employeeId);
}

File: react-frontend/src/service/EmployeeService.js
import axios from 'axios';

const EMPLOYEE_SERVICE_BASE_URL = "http://localhost:9191/employee-service/employees/";
const EMPLOYEE_ID = 1;

class EmployeeService {
    getEmployee() {
        return axios.get(EMPLOYEE_SERVICE_BASE_URL + EMPLOYEE_ID);
    }
}

export default new EmployeeService;